db/structure.sql

```diff
@@ -2142,7 +2140,8 @@ CREATE TABLE public.test_groups (
     updated_at timestamp without time zone NOT NULL,
     assessment_id bigint NOT NULL,
     autotest_settings json DEFAULT '{}'::json NOT NULL,
-    "position" integer NOT NULL
+    "position" integer NOT NULL,
+    tester_index integer
```

Comment:
@mishaschwartz I must have missed this, but... I'm not sure what's going on with this addition? Is `master` out of date, or is this a leftover from other changes you were working on?